libchannelserver/ChannelRPCServer.cpp

```diff
@@ -602,9 +609,14 @@ void dev::ChannelRPCServer::asyncPushChannelMessageHandler(
         CHANNEL_LOG(DEBUG) << "async push channel message" << LOG_KV("seq", channelMessage->seq())
                            << LOG_KV("type", channelMessage->type())
                            << LOG_KV("datalen", channelMessage->length());
+
         asyncPushChannelMessage(toTopic, channelMessage,
-            std::function<void(dev::channel::ChannelException, dev::channel::Message::Ptr,
-                dev::channel::ChannelSession::Ptr)>());
+            [channelMessage](dev::channel::ChannelException e, dev::channel::Message::Ptr,
```

Comment:
add if(m_callback) to avoid another one meet the same problem

Reply:
Use LOG_DESC()  to add pure comments, same as other comments below.